enterprise/system-requirements.md

```diff
@@ -24,9 +24,9 @@ JDK 8
 
 ### Java (J2EE) Application Servers
 
-* Tomcat 7+
-* Jetty 8+
-* WebSphere Application Server (WAS) 8+
+* Apache Tomcat 8.0.42 or later
```

Comment:
I finally found that piece of information I was looking for before. It was tomcat, not jetty.

We can't just say 8.0.42 or later, because certain versions of 8.5 won't work.

The tomcat bug is here: https://bz.apache.org/bugzilla/show_bug.cgi?id=60688#c5

And the information it has about fixes is: 

```
Fixed in:
- trunk for 9.0.0.M18 onwards
- 8.5.x for 8.5.12 onwards
- 8.0.x for 8.0.42 onwards
- 7.0.x for 7.0.76 onwards
```